dist/helpers/chem.js

```diff
@@ -839,6 +843,22 @@ const ExtractJcamp = source => {
     features = extrFeaturesCylicVolta(jcamp, layout, peakUp);
   } else {
     features = extrFeaturesNi(jcamp, layout, peakUp, spectra);
+    if (_format.default.isDSCLayout(layout)) {
+      const {
+        info
+      } = jcamp;
```

Comment:
If jcamp is undefined. It can give TypeError as undefined.

Reply:
@haditariq jcamp is always defined. Therefore, it doesn't need to checkout jcamp is undefined